test-plugins/org.yakindu.sct.generator.c.test/gtests/InternalEventLifeCycle_Naming/InternalEventLifeCycleTest.cc

```diff
@@ -32,14 +32,14 @@ TEST_F(InternalEventLifeCycleTest, InternalEventLifeCycleTest) {
 	intlEvent_enter(&statechart);
 	EXPECT_TRUE(intlEvent_isStateActive(&statechart, intlEvent_r1_A));
 	EXPECT_TRUE(intlEvent_isStateActive(&statechart, intlEvent_r2_C));
-	intlEvent_raise_e(&statechart);
+	intlEvent_Iface_raise_e(&statechart);
```

Comment:
Default naming is 'stateMachineIface_raise_e'.  The prefix (bad name) should simply substitute the state machine name.  So the function name should be 'intlEventIFace_raise_e' if the prefix 'intlEvent' is specified. The user can specify 'intEvent_' if he wants 'intlEvent_Iface_raisee'. So do not add the separator '_' by default.

Reply:
Yes :+1: